gnc/navigator_perception/nodes/stereo_point_cloud_driver.cpp

```diff
@@ -165,13 +220,11 @@ namespace fs = boost::filesystem;
                                                 false, false, leftRightStep);
 
     // Feature detectors for left and right image
```

Comment:
This main function should really be broken up into separate functions. 300 line functions aren't good practice.